.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,34 @@
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Integration tests
+on: [release, pull_request]
```

Comment:
You will almost certainly want to add `master` here to make sure tests pass on merge. The lack of any builds yet is because of this.

Reply:
Thanks Phil.